katalog/thanos/README.md

```diff
@@ -0,0 +1,129 @@
+# Thanos
+
+Basically Thanos is an opensource Prometheus setup that allows to have 2 important features:
+
+1. high availability on Prometheus (setting more prometheus replicas)
+2. long term storage capacity relying on an external object storage
+
+This is the simplified diagram of a Thanos architecture:
+
+![thanos-architecture](images/thanos-arch.png)
+
+# Image repository and tag
+
+* Thanos components versions: `v0.12.2`
+* Thanos components image: `quay.io/thanos/thanos:v0.12.2`
+
+# Setup in Fury Module
+
+We designed 2 different possible setup accordingly with the 2 features listed above:
+
```

Comment:
here you mention that are 2 options and then 3 things are listed, it's confusing IMHO

Reply:
Hi @lzecca78 , you marked this conversation as resolved but I don't see any change in the text. What was the solution?